components/core/src/clp/ffi/ir_stream/protocol_constants.hpp

```diff
@@ -12,8 +13,10 @@ constexpr int8_t LengthUByte = 0x11;
 constexpr int8_t LengthUShort = 0x12;
 
 constexpr char VersionKey[] = "VERSION";
-constexpr char VersionValue[] = "0.0.2";
-constexpr char BetaVersionValue[] = "0.1.0-beta.1";
+constexpr std::string_view VersionValue{"0.1.0"};
+constexpr std::string_view MinimumSupportedVersion{"0.1.0-beta.1"};
+// This is used for IR stream format that predates the key-value pair IR format.
```

Comment:
```suggestion

// This is used for the IR stream format that predates the key-value pair IR format.
```

Notice the preceding empty line before the comment; let's try to follow this practice to follow in Junhao's footsteps.